app/helpers/tools_helper.rb

```diff
@@ -1,12 +0,0 @@
-module ToolsHelper
-  def render_tool(tool)
-    partial = "tools/#{tool.slug.underscore}"
-    render partial, locals: { tool: tool } if partial_exists?(partial)
```

Comment:
Heads up — removing this guard means we'll try (and fail) to render tools that exist in the db but don't have an associated partial.

I think ideally we shouldn't have tools like that. And I think we likely don't. But thought it was worth calling out.